DeadmansSwitch.sol

```diff
@@ -2,34 +2,32 @@
 pragma solidity ^0.8.0;
 
 contract DeadmansSwitch {
-    // TODO: Declare state variables
-    // Hint: You'll need variables for the owner, beneficiary, and last check-in block
+    address public owner;
+    address public beneficiary;
+    uint256 public lastCheckInBlock;
 
-    // TODO: Implement constructor
     constructor(address _beneficiary) {
-        // Hint: Initialize state variables
+        owner = msg.sender;
+        beneficiary = _beneficiary;
+        lastCheckInBlock = block.number;
     }
 
-    // TODO: Implement still_alive function
     function still_alive() public {
-        // Hint: Update the last check-in block
+        require(msg.sender == owner, "Only the owner can call this.");
+        lastCheckInBlock = block.number;
     }
 
-    // TODO: Implement release funds function
     function releaseFunds() public {
-        // Hint: Check if 10 blocks have passed since last check-in
-        // If so, transfer the contract balance to the beneficiary
+        require(block.number > lastCheckInBlock + 10, "Owner is still active.");
```

Comment:
there can be possibility that Owner is dead but enough block has not passed. You should include that in message